### DISCUSSION TITLE
关于123云盘开启了防盗链功能

### DISCUSSION THREAD

**Discussion by aoaoemoji** [AlistGo/alist#4703] [General]
由于之前都是解析的Alist 网盘数量还比较多 搬家属实麻烦 希望可以解决这个问题 很多用户都被防盗链阴间到了 吧 哈哈哈！

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/4704#issuecomment-1621724979

**Comment by aoaoemoji** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by ampcr** [comment]
我就说这个网盘不灵了

**Comment by huolanyun** [comment]
这个网盘最新修改加功能频繁，个人觉得 应该停止支持一段时间，等他修改完了，再支持，毕竟稳定了才好对接

**Comment by aoaoemoji** [comment]
我直接放弃了 把所有资源全部分享了一遍
&nbsp;


741500926
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Garden ***@***.***&gt;; 
发送时间: 2023年7月13日(星期四) 下午5:25
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 关于123云盘开启了防盗链功能 (Discussion #4703)





 
这个网盘最新修改加功能频繁，个人觉得 应该停止支持一段时间，等他修改完了，再支持，毕竟稳定了才好对接
